charts/airflow/docs/guides/quickstart.md

```diff
@@ -59,10 +59,10 @@ We recommend that you start your `custom-values.yaml` file from one of our sampl
 > 
 > The following links should help you extend your `custom-values.yaml` to suit your needs:
 >
-> - [`Docs: Key Features`](../..#key-features)
-> - [`Docs: Frequently Asked Questions`](../..#frequently-asked-questions)
-> - [`Docs: Examples`](../..#examples)
-> - [`Docs: Helm Values`](../..#helm-values)
+> - [`Docs: Key Features`](../../README.md#key-features)
+> - [`Docs: Frequently Asked Questions`](../../README.md#frequently-asked-questions)
+> - [`Docs: Examples`](../../README.md#examples)
+> - [`Docs: Helm Values`](../../README.md#helm-values)
```

Comment:
Same with these, we can remove the `README.md` part, as long as there is a `/` before the `#`.